### ISSUE TITLE
3dUnifize sets qform/sform codes to 3 (Talairach aligned)

### ISSUE THREAD

**Issue by effigies** [afni/afni#65]
```Shell
#!/bin/bash
echo "Run 3dUnifize"
3dUnifize -clfrac 0.4 -prefix out_file.nii.gz -T2 -input in_file.nii.gz

echo; echo "Input file qform/sform"
fslhd in_file.nii.gz | grep -A 14 qform_name

echo; echo "Output file qform/sform"
fslhd out_file.nii.gz | grep -A 14 qform_name
```

Output:
```
Run 3dUnifize
++ 3dUnifize: AFNI version=Debian-16.2.07~dfsg.1-5~nd16.04+1 (Jun 12 2017) [64-bit]
 + Pre-processing: IADVUWI
++ Output dataset ./out_file.nii.gz
++ ===== Elapsed = 0.3 sec

Input file qform/sform
qform_name     Scanner Anat
qform_code     1
qto_xyz:1      -3.125000  0.000000  -0.000000  97.931519
qto_xyz:2      0.000000  3.125000  -0.000000  -68.019211
qto_xyz:3      0.000000  0.000000  4.000000  -106.650566
qto_xyz:4      0.000000  0.000000  0.000000  1.000000
qform_xorient  Right-to-Left
qform_yorient  Posterior-to-Anterior
qform_zorient  Inferior-to-Superior
sform_name     Aligned Anat
sform_code     2
sto_xyz:1      -3.125000  0.000000  0.000000  98.909752
sto_xyz:2      0.000000  3.125000  0.000000  -110.716141
sto_xyz:3      0.000000  0.000000  4.000000  -51.800713
sto_xyz:4      0.000000  0.000000  0.000000  1.000000

Output file qform/sform
qform_name     Talairach
qform_code     3
qto_xyz:1      -3.125000  -0.000000  -0.000000  98.909752
qto_xyz:2      0.000000  3.125000  -0.000000  -110.716141
qto_xyz:3      -0.000000  0.000000  4.000000  -51.800713
qto_xyz:4      0.000000  0.000000  0.000000  1.000000
qform_xorient  Right-to-Left
qform_yorient  Posterior-to-Anterior
qform_zorient  Inferior-to-Superior
sform_name     Talairach
sform_code     3
sto_xyz:1      -3.125000  -0.000000  -0.000000  98.909752
sto_xyz:2      -0.000000  3.125000  -0.000000  -110.716141
sto_xyz:3      0.000000  0.000000  4.000000  -51.800713
sto_xyz:4      0.000000  0.000000  0.000000  1.000000
```

* [Input file](https://drive.google.com/file/d/0B4dQbkKCyJ6uUGdGSE8tZ2VGaGM/view?usp=sharing)
* [Directory with script, input file and output file](https://drive.google.com/drive/folders/0B4dQbkKCyJ6uU1UxbFRtUlZUVUU?usp=sharing)

3dUnifize should not, as far as I understand, modify the qform or sform, so this seems to me likely to be a bug.

**Comment by afni-rickr** [comment]
Because some software (I forget which) was using code=2 to imply standard space, as shown in your sform_code, and because we encountered a lot of it at the time, we changed that to be the default behavior, controllable by an environment variable (AFNI_NIFTI_VIEW).
Whether 3dUnifize (or any other program) should modify any aspect of the header is more general question.  AFNI does not use the NIFTI data format natively.  Therefore there is an interpretation step converting the NIFTI format to AFNI for processing, and then back to NIFTI on write.  NIFTI is an exchange format, so I would expect similar alterations to be modestly common across software packages.
